pennylane/devices/default_clifford.py

```diff
@@ -99,27 +116,50 @@ def observable_stopping_condition(obs: qml.operation.Operator) -> bool:
     return obs.name in _MEAS_OBSERVABLES
 
 
-def _import_stim():
-    """Import stim."""
+def _pl_to_stim(op):
+    """Convert PennyLane operation to a Stim operation"""
     try:
-        # pylint: disable=import-outside-toplevel, unused-import, multiple-imports
-        import stim
-    except ImportError as Error:
-        raise ImportError(
-            "This feature requires stim, a fast stabilizer circuit simulator. "
-            "It can be installed with:\n\npip install stim"
-        ) from Error
-    return stim
+        stim_op = _GATE_OPERATIONS[op.name]
+    except KeyError as e:
+        raise qml.DeviceError(
+            f"Operator {op} not supported on default.clifford and does not provide a decomposition."
+        ) from e
+    return stim_op, " ".join(map(str, op.wires))
+
+
+def _convert_op_to_linear_comb(meas_op):
```

Comment:
It seems that you have `pl_to_stim` for converting operators and  `_convert_op_to_linear_comb` for converting observables? If it is the case, I suggest using a more intuitive name here. Even private functions should have intuitive/consistent names and documentations for easier maintenance and improvement in the future.

Reply:
I originally had a bigger conversion routine for handling all types and classes of PennyLane observables, and so this method name has been a remnant of it. As requested, I have changed the name for both of the functions and attempted to make it more intuitive, and I feel docstrings summarizes their functionality well enough. However, if there are any other major concerns regarding docstrings and function names as such, I would be happy to deal with them and improve upon them in a separate PR.